src/Controllers/controller.ts

```diff
@@ -0,0 +1,105 @@
+import {requestConstructor} from "../Services/Requests/requests";
+import axios from "axios";
+import {photosJSON, roversJSON} from "../Services/Responses/JSONResponseFunctions";
+import {photosHTML, roversHTML} from "../Services/Responses/HTMLResponseFunctions";
+import {Request, Response} from "express";
+
+export const test = (req: Request, res: Response) => {

```

Comment:
in general we have a controller for each resource type - test will live in `testController`
all the endpoints related to rovers in `roversController` etc